docs/source/getting-started/installation.rst

```diff
@@ -300,28 +335,38 @@ increases before proceeding.
 Quick alternative: trying in Docker
 ------------------------------------------------------
 
-As a **quick alternative to installing SkyPilot on your laptop**, we also provide a Docker image with SkyPilot and
-its AWS and GCP dependencies installed for users to quickly try out SkyPilot. You can simply run:
+As a **quick alternative to installing SkyPilot on your laptop**, we also
+provide a Docker image with SkyPilot main branch automatically cloned.
+You can simply run:
 
 .. code-block:: console
 
-  $ docker run -td --name sky --rm -v "$HOME/.sky:/root/.sky:rw" -v "$HOME/.aws:/root/.aws:rw" -v "$HOME/.config/gcloud:/root/.config/gcloud:rw" berkeleyskypilot/skypilot:latest
+  $ # NOTE: '--platform linux/amd64' is needed for Apple silicon Macs
+  $ docker run --platform linux/amd64 \
+    -td --rm --name sky -e SKYPILOT_DEBUG=0 \
+    -v "$HOME/.sky:/root/.sky:rw" \
+    -v "$HOME/.aws:/root/.aws:rw" \
+    -v "$HOME/.config/gcloud:/root/.config/gcloud:rw" \
+    berkeleyskypilot/skypilot-debug
```

Comment:
Should we update this to `skypilot-nightly`?

Reply:
Done.